releasenotes/notes/add-pip-install-options-1d99d61f24e8ee15.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    adds `install_options` field to `riot.Venv`. This field can be used to pass options to `pip install` commands.
```

Comment:
we should be able to handle these via `env` as well, https://pip.pypa.io/en/stable/topics/configuration/#environment-variables